examples/platforms/efr32/src/openthread-core-efr32-config.h

```diff
@@ -167,4 +167,14 @@
 #define OPENTHREAD_CONFIG_HEAP_EXTERNAL_ENABLE 0
 #endif
 
+/**
+ * @def OPENTHREAD_CONFIG_UART_TX_FLUSH_TIMEOUT_MS
+ *
+ * Maximum time to wait for a flush to complete in otPlatUartFlush().
+ *
+ * Value is in milliseconds
+ *
+ */
+#define OPENTHREAD_CONFIG_UART_TX_FLUSH_TIMEOUT_MS 500
```

Comment:
Just curious - in what cases do we need to rely on the timeout?

What about renaming to `OPENTHREAD_CONFIG_EFR32_UART_TX_FLUSH_TIMEOUT_MS` to indicate that this is an EFR32-specific config?

Reply:
@lmnotran please make this change.

Reply:
@jwhui The timeout should realistically never be hit. We just added it so that, just in case something bad happened to the UART, we wouldn't lock up the whole node in an infinite loop